packages/plugin-core/src/_extension.ts

```diff
@@ -620,8 +619,6 @@ export async function _activate(
       // --- Get Version State
       const wsRoot = wsImpl.wsRoot;
       const wsService = new WorkspaceService({ wsRoot });
-      const wsMeta = wsService.getMeta();
-      previousWorkspaceVersion = wsMeta.version;
```

Comment:
This is needed. we are tracking `workspaceState` in `wsMeta` since the StateService changes.